deploy-manage/manage-spaces-fleet.md

```diff
@@ -0,0 +1,92 @@
+---
+products:
+  - id: kibana
+  - id: elastic-agent
+applies_to:
+  stack: preview 9.1
```

Comment:
@kpollich is this feature GA at 9.1?

Reply:
Yes this will be GA in 9.1, though it requires an opt-in and is not enabled by default.